Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # PostgreSQL driver
-gem 'pg'
+gem 'pg', '0.20'
```

Comment:
No worries, that's what I  thought. I also had some issues with my machine and had to change my local ruby version.  I merged the branch earlier so all set!